metacatalog/command_line.py

```diff
@@ -98,6 +98,12 @@ def main():
     migration_parser.add_argument('--message', '-m', type=str, help="Optional revision message. Only used for revision command")
     migration_parser.set_defaults(func=migrate)
 
+    # check extensions for CLI registers
+    from metacatalog.ext import EXTENSIONS
+    for extension in EXTENSIONS.values():
+        if hasattr(extension, 'init_cli'):

```

Comment:
This always evaluates `True`, as class `MetacatalogExtensionInterface` always has the classmethod `init_cli()`.  
I changed the line to `if hasattr(extension, 'cli')`, but this does not resolve the error.

Reply:
Yeah, that was intended behavior, as the `init_cli` is **not** an `ab.cabstractclassmethod`. Thus, you can implement an interface, that does not have a `init_cli` method.
A `cli` method is not a thing, and btw. not a very useful name.